CHANGLOG.md

```diff
@@ -1,3 +1,35 @@
+## v2.0.0 / 2022-04-18
+
+### FEATURES
+
+- Support to route the specified notifications to the specified receivers using `Router` CRD (#121). @wanjunlei
+- Support to mute specified notifications for a given time using `Silence` CRD (#121). @wanjunlei
+- Add support for sending notifications to Feishu (#124). @wanjunlei
+- Support the dynamic modification of the template (#123). @wanjunlei
+- Add support for e2e testing using GitHub action (#111). @zhu733756
```

Comment:
Add support for e2e testing using GitHub action
=>
Add e2e tests with GitHub action